### DISCUSSION TITLE
能否加个选项屏蔽掉视频内的关注三连弹出提示

### DISCUSSION THREAD

**Discussion by qaqz111** [the1812/Bilibili-Evolved#1332] [功能建议 / Ideas]
class 为 `bilibili-player-video-popup-item` `bilibili-player-video-popup-guide-attention` `bilibili-player-video-popup-guide-all` 的一个 div 元素，会遮挡播放中视频的一部分区域，貌似没找到地方可以关，应该在css里注入一段`.bilibili-player-video-popup-guide-attention{display:none!important}`就可以。不知道是否已经有这个选项了我没找到？

顺手贴个测试的视频，22秒处弹出：
https://www.bilibili.com/video/BV1ct411Q7Q6/

**Comment by the1812** [comment]
现在还没有, 会尽快加上

**Comment by Geeyun-JY3** [comment]
最近新出了个投票的，选择后会自动发弹幕（话说这玩意主题色要适配吗）：
![image](https://user-images.githubusercontent.com/49396421/105634254-1e38a200-5e98-11eb-8fe4-2c6b9b1d2976.png)

**Reply by the1812** [reply]
进度条可以换主题色, 所以这个也要屏蔽吗

**Reply by Geeyun-JY3** [reply]
我没需求啊，补充一下弹窗种类而已

**Reply by Geeyun-JY3** [reply]
原来选择的那个选项外面那个圈已经跟随主题色了

**Reply by qaqz111** [reply]
可以给个选项让用户自己选择是否屏蔽，看到一半弹个框出来挡着简直太反人类了

**Reply by Geeyun-JY3** [reply]
> 可以给个选项让用户自己选择是否屏蔽，看到一半弹个框出来挡着简直太反人类了

https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.11.16-preview
> 新增功能 `删除视频弹窗`

功能就是有开关的

**Reply by Geeyun-JY3** [reply]
这个主题色适配了没有（为什么我记忆中好像适配过）

**Reply by the1812** [reply]
还没遇到过(

**Reply by Geeyun-JY3** [reply]
那我给个例子🌚？
https://www.bilibili.com/video/av373871938?start_progress=232456

**Reply by Geeyun-JY3** [reply]
https://www.bilibili.com/read/cv10258099
这玩意官方叫「投票组件」（创作中心中叫「投票弹幕」）。写进相关文档说明里吧。

**Comment by Geeyun-JY3** [comment]
又多一个预约卡片：https://www.bilibili.com/read/cv11283939#:~:text=%E7%9A%84%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%EF%BC%8C-,%E9%A2%84%E7%BA%A6%E5%8D%A1%E7%89%87,-%E5%AE%8C%E5%85%A8%E6%94%AF%E6%8C%81%E7%A8%BF%E4%BB%B6
夜间模式相关不知道要不要适配，懒得找例子了

**Reply by the1812** [reply]
估计得适配, 反正遇到了再说
屏蔽是肯定要屏蔽的

**Reply by Geeyun-JY3** [reply]
已确认，这个预约卡片的 `🔔预约` 跟随主题色要适配。

除了这个和上面提到的投票，同为弹窗的关联视频跟随主题色也需适配：https://github.com/the1812/Bilibili-Evolved/issues/1289#issuecomment-768001001

**Comment by 1970633640** [comment]
我没有抓包看，不知道能不能实现不投票就看投票结果，b站这个做法也是奇怪，会导致短时间内大量低质量弹幕

**Reply by Geeyun-JY3** [reply]
关联：#1754

**Comment by Geeyun-JY3** [comment]
又来一个 `评分弹幕`：
啊B总能给我整出些新花样：https://www.bilibili.com/read/cv12088819

**Comment by Geeyun-JY3** [comment]
官方做了个总结：https://www.bilibili.com/read/cv13543699

**Comment by Geeyun-JY3** [comment]
原来投票弹窗优化过
在某个视频点击过投票弹窗的「❌」后，下次再遇到有投票弹窗稿件，会缩到左下角。
这个东西目前`删除视频弹窗` 功能不会删掉。需要强迫症把这个缩小后的弹窗也干掉吗？
![image](https://user-images.githubusercontent.com/49396421/143849944-80b4a745-153a-4fe3-a8ec-5c49052686d8.png)

**Reply by the1812** [reply]
应该也要, 虽然一开始就屏蔽了的人也不会遇到这个(

**Reply by Geeyun-JY3** [reply]
确实（
